compare_signed_unsigned.py

```diff
@@ -87,22 +110,7 @@ def compare_same_basename(basename: str, locations: List[str]) -> TestResults:
             res.errors += 1
             continue
 
-        if Path(d).is_absolute():
-            # Drop the initial '/'
-            output_dir = Path("reproducible_images", *Path(d).parts[1:])
-        else:
-            output_dir = Path("reproducible_images", d)
-
-        Path.mkdir(output_dir, parents=True, exist_ok=True)
-        output_file = output_dir / basename
-
-        # Erase the signature and other variables before checksumming
-        logging.debug("EraseVariables %s -> %s", src_path, output_file)
-        chksum = sof_ri_info.EraseVariables(src_path, parsed_fw, output_file)
-        assert (
-            chksum is not None
-        ), "this requires sof_ri_info.py version 1e4236be68f7b or above"
-
+        chksum, output_file = checksum_parsed_fw(src_path, parsed_fw)
         dirs_chksum[d] = (chksum, output_file)
```

Comment:
@marc-hb Can we print the `chksum` for each file? It's helpful for debugging the checksum difference.

Reply:
Another good idea, can you submit a follow-up commit?

Reply:
Sure.